lib/google/ads/google_ads/interceptors/logging_interceptor.rb

```diff
@@ -37,7 +37,11 @@ class LoggingInterceptor < GRPC::ClientInterceptor
             customer_user_access.email_address|
             customer_user_access_invitation.email_address|
             change_event.user_email|
-            feed.places_location_feed_data.email_address
+            feed.places_location_feed_data.email_address|
+            local_services_lead.contact_details.phone_number|
+            local_services_lead.contact_details.email|
```

Comment:
Should this be `email_address`?

https://developers.google.com/google-ads/api/reference/rpc/v15/ContactDetails